.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
We recently removed this, for now should add it to your private .git/info/exclude file.

Reply:
Ok

Reply:
unresolving, @wismill please push the fixed commits first, then mark as resolved, thanks. Too hard to keep track of things otherwise.

Reply:
Done. Since part of this MR was merge, I thought it would be closed as it happened to me in other project. Sorry for the noise ;-)